std_msgs/Header header

string finger # one of ff/mf/rf/lf/th

float32 strength